docs/website/docs/building-from-source/riscv.md

```diff
@@ -0,0 +1,125 @@
+# RISC-V cross-compilation
+
+Running on a platform like RISC-V involves cross-compiling from a _host_
+platform (e.g. Linux) to a _target_ platform (a specific RISC-V CPU architecture
+and operating system):
+
+* IREE's _compiler_ is built on the host and is used there to generate modules
+  for the target
+* IREE's _runtime_ is built on the host for the target. The runtime is then
+  pushed to the target to run natively.
+
+## Prerequisites
+
+### Host environment setup
+
+You should already be able to build IREE from source on your host platform.
+Please make sure you have followed the [getting started](./getting-started.md)
+steps.
+
+### Install RISC-V cross-compile toolchain and emulator
+
+You'll need a RISC-V LLVM compilation toolchain and a RISC-V enabled QEMU
+emulator.
+
+See instructions in the following links
+
+* [Clang getting started](https://clang.llvm.org/get_started.html)
+* [RISC-V GNU toolchain](https://github.com/riscv/riscv-gnu-toolchain)
+* [QEMU](https://github.com/qemu/qemu)
+* [RISC-V Linux QEMU](https://risc-v-getting-started-guide.readthedocs.io/en/latest/linux-qemu.html)
+
+!!! note
+    An environment variable `RISCV_TOOLCHAIN_ROOT` needs
+    to be set to the root directory of the installed GNU toolchain. The variable
+    can be used in building the RISCV target and a LLVM AOT module.
+
+### Install Prebuilt RISC-V Tools (RISC-V 64-bit Linux toolchain)
+
+Execute the following script to download the prebuilt RISC-V toolchain and QEMU
+from the IREE root directory:
+
+```shell
+./build_tools/riscv/riscv_bootstrap.sh
+```
+!!! note
+    You also need to set `RISCV_TOOLCHAIN_ROOT`
+    (default at ${HOME}/riscv/toolchain/clang/linux/RISCV).
```

Comment:
This is already mentioned just a few lines above. Omit here?

Reply:
Done.